7주차/[Lv.2]_테이블_해시_함수/최원석.js

```diff
@@ -0,0 +1,12 @@
+function solution(datas, col, row_begin, row_end) {
+  return datas
+    .sort((a, b) => (a[col - 1] === b[col - 1] ? b[0] - a[0] : a[col - 1] - b[col - 1]))
```

Comment:
```suggestion
    .sort((a, b) => a[col - 1] - b[col - 1] || b[0] - a[0])
```

정렬 부분을 더 간결하게 표현할 수 있을 것 같습니다! 😊